src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocstyle/InputJavadocStyleDefaultSettingsFive.java

```diff
@@ -0,0 +1,37 @@
+/*
+JavadocStyle
+scope = (default)private
+excludeScope = (default)null
+checkFirstSentence = (default)true
+endOfSentenceFormat = (default)([.?!][ \t\n\r\f<])|([.?!]$)
+checkEmptyJavadoc = (default)false
+checkHtml = (default)true
+tokens = (default)ANNOTATION_DEF, ANNOTATION_FIELD_DEF, CLASS_DEF, CTOR_DEF, \
+         ENUM_CONSTANT_DEF, ENUM_DEF, INTERFACE_DEF, METHOD_DEF, PACKAGE_DEF, \
+         VARIABLE_DEF, RECORD_DEF, COMPACT_CTOR_DEF
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocstyle;
+
+public class InputJavadocStyleDefaultSettingsFive
+{
+   /**
+    * {@return a string} // ok, foo one
```

Comment:
>// ok, foo one

Please remove it. Or add javadoc that was in issue. It is good practice to show new behavior on what is reported in issue.

Reply:
All comment text that try to explain violation in javadoc , better to place outside of javadoc

Reply:
```
   /**
    * {@return a string}
    */
    public String foo() // ok, foo one
    {
        return "Hello, world";
    }
```
is this fine?